evalml/pipelines/time_series_classification_pipelines.py

```diff
@@ -40,8 +40,24 @@ def fit(self, X, y):
 
         Returns:
             self
+
+        Raises:
+            ValueError: If the number of unique classes in y are not appropriate for the type of pipeline.
         """
         X, y = self._convert_to_woodwork(X, y)
+
```

Comment:
Isn't this exactly the same as `ClassificationPipeline.fit()`?  I feel like we can actually just get rid of the time series fit, unless there's currently a compelling reason to override it with the same thing? @freddyaboulton , did you have something in mind?